src/main/java/io/vertx/core/http/package-info.java

```diff
@@ -1527,6 +1527,21 @@
  * When using a WebSocket as a write stream or a read stream it can only be used with WebSockets connections that are
  * used with binary frames that are no split over multiple frames.
  *
+ * === Using a HTTP proxy for HTTPS connections
+ *
+ * The http client supports accessing https server via a HTTPS proxy (HTTP CONNECT method), e.g. Squid. This is supported
+ * for HTTP/1.x and HTTP/2.
```

Comment:
clarify that only HTTP/1 is supported to the proxy but it can connect to HTTP/1.x and HTTP/2 hosts